alarms/alarm_manager.py

```diff
@@ -155,22 +163,65 @@ def trigger_pokemon(self, pkmn):
 			if config['GEOFENCE'].contains(lat,lng) is not True:
 				log.info(name + " ignored: outside geofence")
 				return
-				
+		
 		#Trigger the notifcations
 		log.info(name + " notication was triggered!")
 		timestamps = get_timestamps(dissapear_time)
-		pkmn_info = {
```

Comment:
Instead of making this an if/else, we could just add when with `.get(key, default)`